helm/kubernetes-prometheus-chart/values.yaml

```diff
@@ -0,0 +1,19 @@
+name: prometheus-operator
+namespace: kube-system
```

Comment:
Just a question. Should the operator run in `kube-system` or in `giantswarm`? As this is where chart-operator runs.

Reply:
I vote for `giantswarm`.

Reply:
yeah me too, I change the default to `giantswarm` (the home of operators)

Reply:
Yes, I really like using `giantswarm`. 👍